Tests/Amr/Advection_AmrLevel/Exec/SingleVortex/inputs

```diff
@@ -41,5 +41,7 @@ amr.plot_int          = 100    # number of timesteps between plot files
 adv.do_tracers = 0
 
 # ERROR TAGGING
-tagging.phierr =  1.01  1.1   1.5
-tagging.max_phierr_lev = 10
+tagging.phierr  =  1.01  1.1   1.5
+tagging.phigrad =  1e+20 1e+20 1e+20
+tagging.max_phierr_lev  = 10
+tagging.max_phigrad_lev = 10
```

Comment:
Since the default of phigrad are huge numbers, I think we should still use `-1` as default for max_phigrad_lev  so that we do not do unnecessary work.